bundles/org.openhab.core.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java

```diff
@@ -59,29 +62,21 @@ public Configuration(final @Nullable Configuration configuration) {
      * @param properties the properties the configuration should be filled. If null, an empty configuration is created.
      */
     public Configuration(Map<String, Object> properties) {
-        this(properties, false);
+        this(properties == null ? emptyMap() : properties, false);
     }
 
     /**
      * Create a new configuration.
      *
-     * @param properties the properties to initialize (may be null)
+     * @param properties        the properties to initialize (may be null)
```

Comment:
Please remove the added spaces between `properties` and `the`

Reply:
The spaces are automatically added by the openhab workspace format settings - the param descriptions are aligned. I think this is intentional?

Reply:
Are you using the most recent IDE setup?
AFAIK this has been an formatter or formatting configuration error and has not been enabled by intention.

Reply:
I used the newest Oomph installer. I just disabled the setting and removed the change.

Reply:
@mlriess Perhaps this will help:
https://github.com/eclipse/smarthome/issues/6424#issuecomment-471728524